tests/end_to_end/test_ip_reputation/data/playbooks/generate_events.yaml

```diff
@@ -0,0 +1,31 @@
+- name: Truncate files
+  hosts: wazuh-manager
+  tasks:
+
+    - name: Truncate file alert.json
+      shell: echo "" > /var/ossec/logs/alerts/alerts.json
+      become: true
+
+- name: Generate events
+  hosts: wazuh-windows
+  tasks:
+
+    - name: "{{ event_description }}"
+      ansible.windows.win_shell: "{{ command }}"
+      # Ignore powershell error
+      ignore_errors: true
```

Comment:
We should ignore specific errors, we could use `failed_when`

Reply:
Done in 584adc0c189079565afb4016ceaf6acf555e4e51